### ISSUE TITLE
Crashes when saving settings and can't connect to RPC endpoint

### ISSUE THREAD

**Issue by ssteiger** [ssteiger/Vyper-Contract-GUI#2]

**Comment by ssteiger** [comment]
Error:

```
"Uncaught (in promise) Error: CONNECTION ERROR: Couldn't connect to node http://127.0.0.1:7545.", source: file:///Users/soeren/github/Vyper-contract-GUI/src/js/assets/web3.min.js (1)
```
Error is happening here:
File ``Vyper-contract-GUI/src/js/init-web3.js``
```
async function setWeb3Provider (httpProvider) {
  if (httpProvider == '') {
    let ganache = 'http://localhost:7545'
    httpProvider = ganache
  }

  web3 = new Web3()
  provider = new Web3.providers.HttpProvider(httpProvider)

  // TODO:
  //      Uncaught (in promise) Error: CONNECTION ERROR: Couldn't connect to
  //      node http://127.0.0.1:7545.",
  //      source: Vyper-contract-GUI/src/js/assets/web3.min.js
  web3.setProvider(provider, function(err, res) {
    if (err) {
      console.log(err)
      toastr.error(err)
    } else {
      console.log(res)
    }
  })
```

How to catch and handle this error and stop electron from crashing...?

**Comment by ssteiger** [comment]
resolved with #a8d880af
